model/atmosphere/diffusion/tests/diffusion_tests/test_diffusion.py

```diff
@@ -428,11 +430,17 @@ def test_run_diffusion_single_step(
     ndyn_substeps,
     backend,
     orchestration,
+    benchmark,
 ):
     if orchestration and not helpers.is_dace(backend):
         pytest.skip("Orchestration test requires a dace backend.")
     if orchestration and data_alloc.is_cupy_device(backend):
         pytest.xfail("GPU compilation fails.")
+
+    if experiment == dt_utils.REGIONAL_EXPERIMENT:
+        # Skip benchmarks for this experiment
```

Comment:
why? (This experiment is not "final" in the sense that we a) want to switch to a MCH production setup instead, which has a smaller (coarser) grid (my understanding), but it might be useful to benchmark a LAM setup as well, because there are quite a lot of boundary stencils that only run in this mode. Of course, it can always be added later...

Reply:
It is just a suggestion, and a way to show how to skip benchmarking for a specific parameter set. I do not think that we need more than one benchmarking per test. Although, if you think that we need to do otherwise, I can change it.